easybuild/easyconfigs/e/EMAN2/EMAN2-2.2-gompi-1.7.20.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock='Binary'
+
+name = "EMAN2"
+version = "2.2"
+
+homepage = 'http://blake.bcm.edu/emanwiki/EMAN2'
+description = """EMAN2 is the successor to EMAN1. It is a broadly based greyscale scientific image 
+ processing suite with a primary focus on processing data from transmission electron microscopes"""
+
+toolchain = {'name': 'gompi', 'version': '1.7.20'}
+
+# no direct download url. 
+# This easyconfig installs the binary distribution (anaconda based) 
+# http://blake.bcm.edu/emanwiki/EMAN2#Download_EMAN2
+sources = ['eman%(version)s.linux64.centos7.sh']
+checksums = ['b07d4fa7a194913b68ab0d3fce4f9c41885634b275be1dadac8f2d1961d9032a']
+
+install_cmd = 'bash %s -f -b -p %%(installdir)s' % sources[0]
+
+# this is required to use the mpi library installed in the cluster
+# http://blake.bcm.edu/emanwiki/EMAN2/Install/BinaryInstallAnaconda#Linux_Clusters
+postinstallcmds = [
+    'export PATH=%(installdir)s/bin:${PATH} && yes | bash %(installdir)s/utils/uninstall_openmpi.sh',
+    'export PATH=%(installdir)s/bin:${PATH} && yes | bash %(installdir)s/utils/build_pydusa_numpy.sh 1.8 --no-test',
+    'export PATH=%(installdir)s/bin:${PATH} && yes | bash %(installdir)s/utils/install_pydusa_numpy.sh 1.8',
+]
+
+sanity_check_paths = {
+    'files': ["bin/%s" % x for x in ["e2version.py", "e2speedtest.py", "e2display.py", "e2proc2d.py"]],
```

Comment:
@boegel I agree that this module should not be combined with others. I instruct my users to don't do it

I don't build it from source because it's a pain in the ass. It has many many dependencies. Also for this kind of apps with so many dependencies I prefer to deploy what developers provide because this is what the developers have tested . I understand your point but IMHO deploying what developers have tested makes things easiers for end users and makes reproducibility easier

I have seen that there is another easyconfig building from source. Feel free to include that one and discard this one